references.bib

```diff
@@ -1,1167 +1,1164 @@
+	Date-Modified = {2019-01-29 01:58:26 +0100},
+	Doi = {10.1145/3182658},
+	Journal = {{ACM} Comput. Surv.},
+	Number = {3},
+	Pages = {51:1--51:39},
+	Timestamp = {Wed, 21 Nov 2018 12:44:28 +0100},
+	Title = {A Survey on Routing in Anonymous Communication Protocols},
+	Url = {https://doi.org/10.1145/3182658},
+	Volume = {51},
+	Year = {2018},
+	Bdsk-Url-1 = {https://doi.org/10.1145/3182658}}
+
+@inproceedings{DBLP:conf/uss/AlexopoulosKT017,
+	Author = {Nikolaos Alexopoulos and Aggelos Kiayias and Riivo Talviste and Thomas Zacharias},
+	Bibsource = {dblp computer science bibliography, https://dblp.org},
+	Biburl = {https://dblp.org/rec/bib/conf/uss/AlexopoulosKT017},
+	Booktitle = {26th {USENIX} Security Symposium, {USENIX} Security 2017, Vancouver, BC, Canada, August 16-18, 2017.},
+	Crossref = {DBLP:conf/uss/2017},
+	Date-Added = {2019-01-29 01:58:04 +0100},
+	Date-Modified = {2019-01-29 01:58:04 +0100},
+	Pages = {1217--1234},
+	Timestamp = {Fri, 01 Sep 2017 12:29:11 +0200},
+	Title = {MCMix: Anonymous Messaging via Secure Multiparty Computation},
+	Url = {https://www.usenix.org/conference/usenixsecurity17/technical-sessions/presentation/alexopoulos},
+	Year = {2017},
+	Bdsk-Url-1 = {https://www.usenix.org/conference/usenixsecurity17/technical-sessions/presentation/alexopoulos}}
+
+@proceedings{DBLP:conf/uss/2017,
+	Bibsource = {dblp computer science bibliography, https://dblp.org},
+	Biburl = {https://dblp.org/rec/bib/conf/uss/2017},
+	Date-Added = {2019-01-29 01:58:04 +0100},
+	Date-Modified = {2019-01-29 01:58:04 +0100},
+	Editor = {Engin Kirda and Thomas Ristenpart},
+	Publisher = {{USENIX} Association},
+	Timestamp = {Fri, 01 Sep 2017 12:24:05 +0200},
+	Title = {26th {USENIX} Security Symposium, {USENIX} Security 2017, Vancouver, BC, Canada, August 16-18, 2017},
+	Url = {https://www.usenix.org/conference/usenixsecurity17},
+	Year = {2017},
+	Bdsk-Url-1 = {https://www.usenix.org/conference/usenixsecurity17}}
+
+@inproceedings{DBLP:conf/asiacrypt/ChaidosFK018,
+	Author = {Pyrros Chaidos and Olga Fourtounelli and Aggelos Kiayias and Thomas Zacharias},
+	Bibsource = {dblp computer science bibliography, https://dblp.org},
+	Biburl = {https://dblp.org/rec/bib/conf/asiacrypt/ChaidosFK018},
+	Booktitle = {Advances in Cryptology - {ASIACRYPT} 2018 - 24th International Conference on the Theory and Application of Cryptology and Information Security, Brisbane, QLD, Australia, December 2-6, 2018, Proceedings, Part {III}},
+	Crossref = {DBLP:conf/asiacrypt/2018-3},
+	Date-Added = {2019-01-29 01:57:14 +0100},
+	Date-Modified = {2019-01-29 01:57:14 +0100},
+	Doi = {10.1007/978-3-030-03332-3\_8},
+	Pages = {191--221},
+	Timestamp = {Thu, 22 Nov 2018 11:41:13 +0100},
+	Title = {A Universally Composable Framework for the Privacy of Email Ecosystems},
+	Url = {https://doi.org/10.1007/978-3-030-03332-3\_8},
+	Year = {2018},
+	Bdsk-Url-1 = {https://doi.org/10.1007/978-3-030-03332-3%5C_8}}
+
+@proceedings{DBLP:conf/asiacrypt/2018-3,
```

Comment:
can you remove the conference entry, as it is less useful that the more specific pointer you provide above the the individual paper within it.